assembl/static2/js/app/router.js

```diff
@@ -0,0 +1,26 @@
+import { useRouterHistory } from 'react-router';
+import { createHistory, useBeforeUnload } from 'history';
+
+import PiwikReactRouter from 'piwik-react-router';
+import get from 'lodash/get';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
Why do have this bootstrap.css import here? Bad copy paste?

Reply:
oh yes, forgot to remove it, thanks